components/KeyManagerPermissions/KeyManagerPermissions.tsx

```diff
@@ -127,6 +127,27 @@ const KeyManagerPermissions: React.FC = () => {
               ),
             )}
           </div>
+          <div className="buttons">
```

Comment:
We have this same code 3 times now for the 3 groups of permissions. Maybe it makes sense to reuse this as a component or something now